### PR TITLE
Implement Home Page Comp #102 Provide Store to Stroybook env #107 

### PR DESCRIPTION
**Pull request by jkpark104**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/team-GIFT/gift/wiki/Convention#commit-convention
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Configuration related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Related Issue Number

- #107  #102 

## What does this PR do?
feat: Implement Home Page Comp 

1. Implement storybook file #102 
2. Implement HomeComp using components

fix: Provide Store to Stroybook env #107 

1. modify .storybook/main.js to fix GIPHY_API_KEY
2. modify .storybook/preview.js to provide store

### CODE REVIEW

File: src/pages/Home/Home.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  const { isLoading: isTrendingGifsLoading, cards: trendingGifs } =
+    useSelector(trendingGifsSelector);
+  const { isLoading: isArtistGifsLoading, cards: artistGifs } =
+    useSelector(artistGifsSelector);
+
+  return (
+    <>
+      {!isTrendingGifsLoading && !isArtistGifsLoading && (
+        <Wrapper>
```
Comment (jkpark104) [inline src/pages/Home/Home.tsx:31]: wrapper header 컴포넌트 빼기

Comment (sosoYim) [inline src/pages/Home/Home.tsx]: Wrapper와 Header를 빼주세요^^

**Review by sosoYim — APPROVED** [review]
Approuved!

File: src/pages/Home/Home.tsx (hunk 1 above)
Comment (sosoYim) [reply src/pages/Home/Home.tsx]: 확인

**Pull request merged**